ext/uri/php_uri.c

```diff
@@ -800,8 +800,7 @@ PHP_METHOD(Uri_Rfc3986_Uri, __serialize)
 	zend_hash_next_index_insert(Z_ARRVAL_P(return_value), &arr);
 
 	/* Serialize regular properties: second array */
-	ZVAL_ARR(&arr, uri_object->std.handlers->get_properties(&uri_object->std));
-	Z_TRY_ADDREF(arr);
```

Comment:
P.S. this was copy-pasted from https://github.com/php/php-src/blob/1fbb6665458cd0639c6a695557f909755129e919/ext/random/engine_mt19937.c#L328

Reply:
Oops, sorry, I had to re-read the description of the PR to realize that adding a reference to refcount has nothing to do with the issue. My bad!